src/js/components/Marathon.jsx

```diff
@@ -178,6 +181,15 @@ var Marathon = React.createClass({
     );
   },
 
+  getChaosModal: function () {
+    return (
+      <ChaosModalComponent
+        isChaosInProgress={this.state.isChaosInProgress}
```

Comment:
This depends currently missing functionality. Basically, the idea is that Marathon (although I'd prefer a dedicated store for this) keeps track of the Chaos state. 

This is because we want to:

- disable the `Chaos` button (used to reveal the chaos modal) in the TabPanesComponent if Chaos is already running
- disable the `Start Chaos` button (used to initiate the chaos function) inside the modal if Chaos is already running